src/frontend/js/widgets/Dashboard/components/DashboardItem/DashboardItemCourseEnrolling.tsx

```diff
@@ -92,6 +93,7 @@ export const DashboardItemCourseEnrolling = ({
   order,
   icon = false,
   notEnrolledUrl = '#',
+  hideEnrollButtons,
```

Comment:
`disableEnrollButton` is a better name WDYT ?

Reply:
I think it could be confusing because disabled could mean "display the button but turn them disabled", but in this scenario we want to completely hide them, nope?

Reply:
My bad! I was confused by `NotEnrolled` and `DashboardItemCourseEnrollingRuns` components.
https://github.com/openfun/richie/pull/2119#discussion_r1371564619